conan/tools/google/bazel.py

```diff
@@ -0,0 +1,49 @@
+import os
+import json
+
+from conan.tools import CONAN_TOOLCHAIN_ARGS_FILE
+from conans.util.files import load
+
+class Bazel(object):
+    def __init__(self, conanfile):
+        self._conanfile = conanfile
+        self._get_bazel_project_configuration()
+
+    def configure(self, args=None):
```

Comment:
If no config step in bazel, you can safely remove this method.

Reply:
As far as I've seen, the convention when writing the `build()` method for a project's `conanfile.py` is:

```
def build(self):
    bazel = Bazel(self)
    bazel.configure()
    bazel.build(whatever)
```

Wouldn't deleting the configure method break this pattern?